src/libs/ascent/runtimes/ascent_expression_eval.cpp

```diff
@@ -879,27 +961,57 @@ initialize_functions()
   field_sig["description"] = "Return a mesh field given a its name.";
 
   //---------------------------------------------------------------------------
-
+  // topo()
+  //---------------------------------------------------------------------------
   conduit::Node &topo_sig = (*functions)["topo"].append();
-  topo_sig["return_type"] = "topo";
-  topo_sig["filter_name"] = "topo";
+  topo_sig["return_type"] = "topology";
+  topo_sig["filter_name"] = "expr_mesh_topology";
```

Comment:
Can this have the same filter name as topolgoy_sig below it (line 977)?

Reply:
yes, think of these as two aliases that map to the same filter. 

However, I typo below -- we need to set params for:

`topology_sig` instead of `topo_sig`, or else `topology_sig` isn't inited. 

The original function name was only `topo`, but I wanted us to be able to call the more explicit `topology` as well.

The return type for both has to be `topology`, b/c that is the new name I deemed moving forward for the expression language object that holds the topology.